### PR TITLE
Created the initial commit for the crawler-db-pull role in the ansibl…

### PR DESCRIPTION
**Pull request by pmaslana**
…e roles repo

### CODE REVIEW

File: crawler-db-pull-on-boot/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+blockchain_backup_bucket: ""
+network: ""
```
Comment (cmmarslender) [inline crawler-db-pull-on-boot/defaults/main.yml:3]: Not really a harm in defaulting this to mainnet

File: crawler-db-pull-on-boot/meta/main.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+---
```
Comment (cmmarslender) [inline crawler-db-pull-on-boot/meta/main.yml:1]: If this is empty we don't need this directory at all

File: crawler-db-pull-on-boot/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+- name: Copy over the crawler db-pull.sh file
+  template:
+    src: "crawler-db-pull.sh.j2"
+    dest: "{{ chia_root }}/db-pull.sh"
```
Comment (cmmarslender) [inline crawler-db-pull-on-boot/tasks/main.yml:5]: Maybe prefix dest file name with crawler too?

File: crawler-db-pull-on-boot/tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  template:
+    src: "crawler-db-pull.service.j2"
+    dest: "/etc/systemd/system/crawler-db-pull.service"
+    owner: root
+    group: root
+    mode: 0755
+
+- name: Enable DB pull service
+  systemd:
+    name: db-pull.service
```
Comment (cmmarslender) [inline crawler-db-pull-on-boot/tasks/main.yml:20]: Crawler prefix is missing

**Review by cmmarslender — CHANGES_REQUESTED** [review]

**Review by austinsirkin — APPROVED** [review]

**Review by cmmarslender — APPROVED** [review]

**Pull request merged**